avail/src/main/kotlin/avail/dispatch/InternalLookupTree.kt

```diff
@@ -869,7 +869,7 @@ internal constructor(
 		// the actually occurring TypeTag will have to be looked up, and if not
 		// found, its ancestors must be searched.
 		// NOTE: Among other things, this can cause elements taking primitive
-		// types (e.g., ANY.o, or even MODULE.o) to be copied down into, say,
+		// types (e.g., ANY.o, or even Module.o) to be copied down into, say,
```

Comment:
This is referring to PrimitiveTypeDescriptor.Types.MODULE, and should remain all uppercase.  Same as line 1045.